src/Microsoft.ML.Transforms/NAIndicatorTransform.cs

```diff
@@ -49,17 +60,14 @@ public sealed class Arguments : TransformInputBase
             public Column[] Column;
         }
 
-        public const string LoaderSignature = "NaIndicatorTransform";
-
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
-                // REVIEW: temporary name
                 modelSignature: "NAIND TF",
                 verWrittenCur: 0x00010001, // Initial
                 verReadableCur: 0x00010001,
                 verWeCanReadBack: 0x00010001,
-                loaderSignature: LoaderSignature,
+                loaderSignature: nameof(NAIndicatorTransform),
```

Comment:
>NAIndicatorTransform [](start = 40, length = 20)

Will changing this break the older models?  Previously it was NaIndicator and now its NAIndicator -- not sure if signatures are case sensitive.

Reply:
Good point, I will need to check this.

---
In reply to: [225364514](https://github.com/dotnet/machinelearning/pull/1217#discussion_r225364514) [](ancestors = 225364514)

Reply:
Actually, I don't like the idea for loader signatures to be `nameof`'s. This way, we can inadvertently break back-compat by renaming.

And BTW we WILL be renaming this to something like NAIndicatorTransformer

---
In reply to: [225367296](https://github.com/dotnet/machinelearning/pull/1217#discussion_r225367296) [](ancestors = 225367296,225364514)